vaadinfrontend/frontend/themes/datamanager/card.css

```diff
@@ -0,0 +1,27 @@
+.card {
+  display: flex;
+  flex: auto;
+  flex-direction: column;
+  border-color: var(--lumo-contrast-10pct);
+  padding: var(--lumo-space-l);
+  font-size: var(--lumo-font-size-s);
+  box-shadow: var(--lumo-box-shadow-s);
+  border-radius: var(--lumo-border-radius-m);
+  box-sizing: border-box;
+  background-color: var(--lumo-base-color);
+  margin: 5px;
+  gap: 1em;
+  overflow: hidden;
```

Comment:
Why is this? I thought it is `flex` and it expands until the content is covered.

Reply:
The card is styled as a flexcomponent, however I'd argue we don't want to allow the card to expand beyond the outer container boundaries, which is why we set the overflow to hidden. 

The background color is set as the default color chosen by vaadin and the gap property allows the card to behave spacing wise similiar to a vaadin layout, which is the current style on which we based the data-manager 

I'm more than open to discuss if the margin property is necessary in the card itself, since it's there to allow for space between the cards if multiple cards are aligned next to each other